data/presets/baking_oven/bakehouse.json

```diff
@@ -0,0 +1,24 @@
+{
+    "icon": "Bakehouse",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "tags": {
+        "amenity": "baking_oven",
+        "cuisine": "Bakehouse"
+    },
+    "reference": {
+        "key": "cuisine"
```

Comment:
reference should be the main tag of the preset, here: `amenity=baking_oven`
```json
        "key": "amenity",
        "key": "baking_oven"
```

this can be completely omitted if the preset only has one tag in `tags` (as I would assume will be the case here).

Reply:
Sir just kept the key = amenity as suggested